site/content/docs/5.3/examples/form/form.js

```diff
@@ -16,6 +16,19 @@
         // Focus on first error
         const invalidItems = form.querySelectorAll(':invalid')
         invalidItems[0].focus()
+        // Add the id of the corresponding invalid message to each invalid field
+        invalidItems.forEach(element => {
+          const linkedLabel = element.getAttribute('aria-labelledby')
+          const closestInvalidFeedback = element.closest('.mb-3').querySelector('.invalid-feedback').getAttribute('id')
+          const valuesArray = [linkedLabel, closestInvalidFeedback].join(' ')
+          element.setAttribute('aria-labelledby', valuesArray)
+        })
+        // Remove the id of the corresponding invalid message to each valid field
+        const validItems = form.querySelectorAll(':valid:not([type=\'submit\'])')
+        validItems.forEach(element => {
+          const closestLabel = element.closest('.mb-3').querySelector('.form-label').getAttribute('id')
```

Comment:
Same as above, can't we select the parent directly ?

Reply:
It was made for better performance but it introduced a problem so I reverted to first version.